Wrappers/Python/cil/optimisation/functions/Function.py

```diff
@@ -79,13 +80,28 @@ def proximal_conjugate(self, x, tau, out = None):
         .. math:: \mathrm{prox}_{\tau F^{*}}(x) = x - \tau\mathrm{prox}_{\tau^{-1} F}(\tau^{-1}x)
                 
         """
+        try:
+            tmp = x
+            x.divide(tau, out = tmp)
```

Comment:
Interesting, why would this happen?

Reply:
Numpy division casts to floats, and then (if specified) the output datacontainer is the wrong type. We have a unittest that uses integers... I'm not sure how useful this is to handle. Maybe the algorithm setup should check got floating types.